data/helpers.d/backend

```diff
@@ -213,3 +213,209 @@ ynh_remove_fpm_config () {
 	ynh_secure_remove "$fpm_config_dir/conf.d/20-$app.ini" 2>&1
 	sudo systemctl reload $fpm_service
 }
+
+# NODEJS HELPERS
+
+n_install_dir="/opt/node_n"
+node_version_path="/opt/node_n/n/versions/node"
+# N_PREFIX is the directory of n, it needs to be loaded as a environment variable.
+export N_PREFIX="$n_install_dir"
+
+# Install Node version management
+#
+# [internal]
+#
+# usage: ynh_install_n
+ynh_install_n () {
+	echo "Installation of N - Node.js version management" >&2
+	# Build an app.src for n
+	mkdir -p "../conf"
+	echo "SOURCE_URL=https://github.com/tj/n/archive/v2.1.7.tar.gz
+SOURCE_SUM=2ba3c9d4dd3c7e38885b37e02337906a1ee91febe6d5c9159d89a9050f2eea8f" > "../conf/n.src"
+	# Download and extract n
+	ynh_setup_source "$n_install_dir/git" n
+	# Install n
+	(cd "$n_install_dir/git"
+	PREFIX=$N_PREFIX make install 2>&1)
+}
+
+# Load the version of node for an app, and set variables.
+#
+# ynh_use_nodejs has to be used in any app scripts before using node for the first time.
+#
+# 2 variables are available:
+#   - $nodejs_path: The absolute path of node for the chosen version.
+#   - $nodejs_version: Just the version number of node for this app. Stored as 'nodejs_version' in settings.yml.
+# And 2 alias stored in variables:
+#   - $nodejs_use_version: An alias to load a node version in the current shell. Especially useful for systemd.
+#     NB: $PATH will contain the path to node, it has to be propagated to any other shell which needs to use it.
+#   - $ynh_node_exec: An alias to execute a command with node.
+#
+# usage: ynh_use_nodejs
+ynh_use_nodejs () {
+	nodejs_version=$(ynh_app_setting_get $app nodejs_version)
+
+	load_n_path="[[ :$PATH: == *\":$n_install_dir/bin:\"* ]] || PATH=\"$n_install_dir/bin:$PATH\"; N_PREFIX="$n_install_dir""
+
+	nodejs_use_version="$n_install_dir/bin/n -q $nodejs_version"
+
+	# "Load" a version of node
+	eval $load_n_path; $nodejs_use_version
+
+	# Get the absolute path of this version of node
+	nodejs_path="$(n bin $nodejs_version)"
+
+	# Make an alias for node use
+	ynh_node_exec="eval $load_n_path; n use $nodejs_version"
+}
+
+# Install a specific version of nodejs
+#
+# n (Node version management) uses the PATH variable to store the path of the version of node it is going to use.
+# That's how it changes the version
+#
+# ynh_install_nodejs will install the version of node provided as argument by using n.
+#
+# usage: ynh_install_nodejs [nodejs_version]
+# | arg: nodejs_version - Version of node to install.
+#        If possible, prefer to use major version number (e.g. 8 instead of 8.10.0).
+#        The crontab will handle the update of minor versions when needed.
+ynh_install_nodejs () {
+	# Use n, https://github.com/tj/n to manage the nodejs versions
+	nodejs_version="$1"
+	local n_install_script="https://git.io/n-install"
```

Comment:
This variable isn't used anywhere.

Reply:
Probably an old line, not used anymore...